bedrock/privacy/templates/privacy/index.html

```diff
@@ -48,9 +48,13 @@ <h2>{{ _('Contact Mozilla') }}</h2>
           <span itemprop="streetAddress">331 E. Evelyn Ave</span>,<br>
           <span itemprop="addressLocality">Mountain View</span>,
           <span itemprop="addressRegion">CA</span>
-          <span itemprop="postalCode">94041</span>
+          <span itemprop="postalCode">94041</span><br>
+          <span itemprop="email"><a href="mailto:compliance@mozilla.com">compliance@mozilla.com</a></span>
         </span>
       </p>
+      {% if l10n_has_tag('privacy_email_2019') %}
+      <p><a href="https://app.onetrust.com/app/#/webform/4ba08202-2ede-4934-a89e-f0b0870f95f0">{{ _('See here for Data Subject Access Requests.') }}</a></p>
```

Comment:
@amychurchwell should avoid using actual URL in localizable string. Let's replace it with a variable. Filing an issue on this to get this fixed.

Reply:
The URL is outside the string; the entire string is inside the link: `{{ _('See here for Data Subject Access Requests.') }}`